src/visualizr/anitalker/config.py

```
@@ -410,0 +1,409 @@
+
+    def make_eval_diffusion_conf(self):
+        return self._make_diffusion_conf(self.T_eval)
+
+    def make_latent_diffusion_conf(self):
+        return self._make_latent_diffusion_conf(self.T)
+
+    def make_latent_eval_diffusion_conf(self):
+        # latent can have different eval T
+        return self._make_latent_diffusion_conf(self.latent_T_eval)
+
+    def make_dataset(self):
```

Ruff (0.12.2)

Missing docstring in public method

(D102)